src/api/app/models/comment.rb

```diff
@@ -75,4 +79,10 @@ def blank_or_destroy
   def destroy
     super
   end
+
+  private
+
+  def check_is_child
+    errors[:type] << "is reserved for storing the inheritance class which was not found" unless type.safe_constantize.try(:superclass) == Comment
```

Comment:
that's a guide, not a law. And if you cross 80-100 characters with it, you have to ignore that part. Your line is 143 chars and that requires quite some scrolling to understand/review.

There are some creative ways to use the post-if without making the line too long, e.g. 
```
def add_type_error
  errors[:type] << "is reserved for storing the inheritance class which was not found"
end

def check_is_child
  add_type_error unless type.safe_constantize.try(:superclass) == Comment
end
```

But I would just do what @bgeuken suggests.